squid/README.md

```diff
@@ -0,0 +1,3 @@
+# squid
+
+An ubuntu 16.04 docker image with squid 3.5 installed.
```

Comment:
Please sir, may I have more documentation sir? (At a minimum, just having an example docker run command of the image would be enough to satisfy me.)